tdrs-frontend/src/actions/sttList.test.js

```diff
@@ -31,11 +31,43 @@ describe('actions/stts.js', () => {
     ])
   })
 
+  it('fetches a list of stts and puts "Federal Government" as the first option if it exists, when the user is authenticated', async () => {
```

Comment:
This is an added test to ensure that `Federal Government` is the first option in the combo box.